javascriptv3/example_code/README.rst

```diff
@@ -0,0 +1,43 @@
+###############################
+AWS SDK for JavaScript version 3 (v3) examples
+###############################
+
+These are examples for the AWS SDK for JavaScript (v3) public documentation.
+
+The prerelease version of the version 3 of the SDK is available `here <at https://github.com/aws/aws-sdk-js-v3>`_ .
+The 'SDK for JavaScript Developer Guide' for V3 is also scheduled for release later in 2020, and the topic containing this example will be hosted
```

Comment:
I don't know that the date should be listed here, unless you have pretty close to 100% confidence.

Reply:
changed to 'pending release' throughout.